src/libstd/sync/mutex.rs

```diff
@@ -147,6 +140,7 @@ pub struct MutexGuard<'a, T: ?Sized + 'a> {
     // disregard field privacy).
     __lock: &'a Mutex<T>,
     __poison: poison::Guard,
+    __inner: parking_lot::MutexGuard<'a, T>,
```

Comment:
Are these underscores still necessary? I think https://github.com/rust-lang/rust/issues/12808 was the relevant issue.